src/ngraph/runtime/intelgpu/CMakeLists.txt

```diff
@@ -16,12 +16,16 @@
 
 set(SRC
     intelgpu_backend.cpp
+    intelgpu_tensor_view.cpp
 )
 
 if (NGRAPH_INTELGPU_ENABLE)
     add_library(intelgpu_backend SHARED ${SRC})
     set_target_properties(intelgpu_backend PROPERTIES VERSION ${NGRAPH_VERSION} SOVERSION ${NGRAPH_API_VERSION})
     target_link_libraries(intelgpu_backend PUBLIC ngraph)
+    find_package(CLDNN REQUIRED)
```

Comment:
Do we need so specify a minimum version?

Reply:
I'm not sure. I didn't find any limitation on the interface. Perhaps we will do that later, if we use a specific, version dependant things.